app/models/metric/statistic.rb

```diff
@@ -0,0 +1,13 @@
+module Metric::Statistic
+  def self.calculate_stat_columns(obj, timestamp)
+    return {} unless obj.respond_to?(:container_groups)
+
+    capture_interval = Metric::Helper.get_time_interval(obj, timestamp)
+    container_groups = ContainerGroup.where("ems_id = #{obj.id} OR old_ems_id = #{obj.id}")
```

Comment:
@zeari I believe the AR way to do this query is:
`ContainerGroup.where(:ems_id => obj.id).or(ContainerGroup.where(:old_ems_id => obj.id))`
/cc @matthewd 
Please refactor in separate PR since I am merging this PR as is.